app/javascript/packs/application.js

```diff
@@ -16,3 +16,7 @@
 // const imagePath = (name) => images(name, true)
 
 console.log('Hello World from Webpacker')
```

Comment:
- [ ] please avoid committing any console.log they make your code slower and harder to read.